src/service/viewer-impl.js

```diff
@@ -856,17 +861,22 @@ export class ViewerImpl {
 }
 
 /**
- * Creates an error for the case where a channel cannot be established.
+ * Creates a dev error for the case where a channel cannot be established.
  * @param {*=} opt_reason
  * @return {!Error}
  */
 function getChannelError(opt_reason) {
+  let channelError;
   if (opt_reason instanceof Error) {
     opt_reason = duplicateErrorIfNecessary(opt_reason);
     opt_reason.message = 'No messaging channel: ' + opt_reason.message;
-    return opt_reason;
+    channelError = opt_reason;
+  } else {
+    channelError = new Error('No messaging channel: ' + opt_reason);
   }
-  return new Error('No messaging channel: ' + opt_reason);
+  // Force convert user error to dev error
+  channelError.message = channelError.message.replace(USER_ERROR_SENTINEL, '');
```

Comment:
This is, imho, too nuanced to spread from the log system. We already export `isUserError` there. Can we also expose something like `stripUserError`?

Reply:
Sure. I agree a helper method would be helpful if there are future use cases. Are you thinking of something like 
```
error.message = stripUserError(error.message)
```

Or something like 
```
error = convertToDevError(error)
```